extension/framework/runner/test_data/gohan_sync_fetch_mock.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2017 NTT Innovation Institute, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var SCHEMA_INCLUDES = [];
+var SCHEMAS = ["./schema.yaml"];
+var PATH = "/v1.0/networks";
+
+function testUnexpectedCall() {
+    result = gohan_sync_fetch("unexpectedKey");
```

Comment:
"result" becomes a global variable. change to `var result = ...` or simply `gohan_sync_fetch("unexpected")`